runtime/compiler/control/CompilationThread.cpp

```diff
@@ -2107,8 +2107,14 @@ bool TR::CompilationInfo::shouldRetryCompilation(TR_MethodToBeCompiled *entry, T
                if (entry->_compilationAttemptsLeft == 1)
                   entry->_doNotUseAotCodeFromSharedCache = true;
                // Intentionally fall through next case
-            case compilationInterrupted:
             case compilationStreamFailure:
+               // This feature TR_RequireJITServer is used when we would like the client to fail when server crashes
```

Comment:
Can this happen at the server as well? If so, we may want to check that we are in client mode.

Reply:
No, i think this function `shouldRetryCompilation()` is only called on the client.